### PR TITLE
Add max_log_len option to control length of printed log

### PR DESCRIPTION
**Pull request by lvhan028**
```shell
lmdeploy serve api_server <model_path> --max-log-len 100
```

### CODE REVIEW

File: lmdeploy/serve/gradio/app.py [hunk 1]
```diff
@@ -34,8 +34,8 @@ def run(model_path_or_server: str,
             config instance. Default to none.
         chat_template_config (ChatTemplateConfig): chat template configuration.
             Default to None.
-        tp (int): tensor parallel for Turbomind
-        share (bool): whether to create a publicly shareable link for the app
+        share (bool): whether to create a publicly shareable link for the app,
+        max_log_len (int):
```
Comment (AllentDan) [inline lmdeploy/serve/gradio/app.py:38]: lack description.

**Comment by irexyc** [comment]
Could we add `filename` and `lineno` to the log like https://github.com/vllm-project/vllm/blob/855c8ae2c9a4085b1ebd66d9a978fb23f47f822c/vllm/logger.py#L19 ?  The current log information cannot determine where it is printed

**Comment by lvhan028** [comment]
> Could we add `filename` and `lineno` to the log like https://github.com/vllm-project/vllm/blob/855c8ae2c9a4085b1ebd66d9a978fb23f47f822c/vllm/logger.py#L19 ? The current log information cannot determine where it is printed

I will pull another request to support it.

**Review by irexyc — APPROVED** [review]

**Review by AllentDan — APPROVED** [review]
LGTM

**Pull request merged**